samples/bluetooth/central_ht/src/main.c

```diff
@@ -0,0 +1,275 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2020 SixOctets Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stddef.h>
+#include <errno.h>
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <sys/byteorder.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+
+static struct bt_conn *default_conn;
+static struct bt_uuid_16 uuid = BT_UUID_INIT_16(0);
+static struct bt_gatt_discover_params discover_params;
+static struct bt_gatt_subscribe_params subscribe_params;
+
+static double power(double x, s8_t y)
```

Comment:
math.h pow function take double, double, should we perhaps put this into minimal-libc in zephyr with that prototype?

Reply:
there is reference to pow in drivers/sensor/light_sensor, from where (library) does this get referenced? If pow already exists/used we should reuse that implementation.

Reply:
@cvinayak It comes from math.h, but libc-minimal zephyrs-builtin does not have it, if using newlibc then it is available:
That is why it looks like this:
```
#if defined(CONFIG_NEWLIB_LIBC)
#include <math.h>

#define GPTP_POW2(exp) pow(2, exp)
```